### ISSUE TITLE
The SEBlock source code looks different from the ECAPA_TDNN paper.

### ISSUE THREAD

**Issue by Sang-Hoon-Pakr** [speechbrain/speechbrain#2075]
### Describe the bug

speechbrain/lobes/models/ECAPA_TDNN.py

------------------------------------------------------------
def __init__(self, in_channels, se_channels, out_channels):
        super(SEBlock, self).__init__()

        self.conv1 = Conv1d(
            in_channels=in_channels, out_channels=se_channels, kernel_size=1
        )
        self.relu = torch.nn.ReLU(inplace=True)
        self.conv2 = Conv1d(
            in_channels=se_channels, out_channels=out_channels, kernel_size=1
        )
        self.sigmoid = torch.nn.Sigmoid()

.... 

### Expected behaviour

speechbrain/lobes/models/ECAPA_TDNN.py

def __init__(self, in_channels, se_channels, out_channels):
        super(SEBlock, self).__init__()

        self.conv1 = Conv1d(
            in_channels=in_channels, out_channels=se_channels, kernel_size=1
        )
        self.relu = torch.nn.ReLU(inplace=True)
       
       # added line 
        self.bn = torch.nn.BatchNorm1d(out_channels)

        self.conv2 = Conv1d(
            in_channels=se_channels, out_channels=out_channels, kernel_size=1
        )
        self.sigmoid = torch.nn.Sigmoid()

**Comment by Sang-Hoon-Pakr** [comment]
@TParcollet Hello 

Thank you for your consideration.

Could you check this ? 
I want to know answer for this...

**Comment by Sang-Hoon-Pakr** [comment]
sorry i saw it wrong
